src/test/scala/com/datastax/gatling/plugin/simulations/cql/BoundCqlTypesSimulation.scala

```diff
@@ -113,12 +118,12 @@ class BoundCqlTypesSimulation extends BaseCqlSimulation {
 
   // A predicate function can be useful when we want to do multiple comparisons on data in a single row
   private def preparedCqlPredicate(row:Row):Boolean =
-    row.getBoolean("boolean_type") && (!row.getString("null_type").equals("test"))
+    row.getBoolean("boolean_type") && row.isNull("null_type")
```

Comment:
don't we need to check `.equals("test"))` any more?

Reply:
The old impl worked because it was going through the old check API which handled null values under the covers.  What we're really interested in testing here is whether null_type is still null after the insert, so it seemed better to just explicitly test for that.